app/src/main/kotlin/com/saltedge/authenticator/tools/ViewTools.kt

```diff
@@ -105,3 +106,29 @@ fun ViewGroup.inflateListItemView(resId: Int): View =
 fun TextView.setTextColorResId(colorResId: Int) {
     ContextCompat.getColor(this.context, colorResId).also { this.setTextColor(it) }
 }
+
+/**
+ * Get display height
+ *
+ * @receiver Activity - Activity object
+ * @return height size
+ */
+fun Activity.getDisplayHeight(): Int {
+    val display = (this.getSystemService(Context.WINDOW_SERVICE) as WindowManager).defaultDisplay
+    val size = Point()
+    display?.getSize(size)
+    return size.y
+}
+
+/**
+ * Get display width
+ *
+ * @receiver Activity - Activity object
+ * @return width size
+ */
+fun Activity.getDisplayWidth(): Int {
```

Comment:
I understand the tests will be unnecessary here or ...?

Reply:
not yet